NextcloudTalk/ContactsTableViewCell.m

```diff
@@ -95,4 +98,16 @@ - (void)setUserStatus:(NSString *)userStatus
     }
 }
 
+- (void)setUSerStatusMessage:(NSString *)userStatusMessage withIcon:(NSString*)userStatusIcon {
+    if (![userStatusMessage isEqualToString:@""] || ![userStatusIcon isEqualToString:@""]) {
+        NSString *statusMessage = [NSString stringWithFormat:@"%@%@", userStatusIcon, userStatusMessage];
```

Comment:
Add a gap of 2 white spaces between userStatusIcon and userStatusMessage
`NSString *statusMessage = [NSString stringWithFormat:@"%@  %@", userStatusIcon, userStatusMessage];`